src/app/ui/bridge/InformationCard.style.ts

```diff
@@ -0,0 +1,13 @@
+'use client';
+
+import type { BoxProps, Breakpoint } from '@mui/system';
+import { styled } from '@mui/system';
+import { BridgePageContainer } from './BridgePage.style';
+
+export const InformationCardContainer = styled(BridgePageContainer, {
+  shouldForwardProp: (prop) => prop !== 'active',
```

Comment:
This does not seem to be needed

Reply:
Good point, I stumble into this pitfall a lot! Will keep it in mind